addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/scanrules/ScriptsActiveScanner.java

```diff
@@ -252,4 +254,23 @@ public int getCweId() {
     public int getWascId() {
         return 0;
     }
+
+    @Override
+    public Map<String, String> getAlertTags() {
+        return Map.of(
```

Comment:
Worth extracting a constant, could also use a stream and map, since it's expected to be enabled on all.